src/client/components/ResultMessageModalWindow/CelebrationImage.component.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import emojiConfettiBall from '../../assets/images/icons/emoji-confetti-ball.svg';
+import dizzyFace from '../../assets/images/icons/emoji-dizzy-face.svg';
+import './ResultMessageModalWindow.styles.css';
+import PropTypes from 'prop-types';
+
+export default function CelebrationImage({ correctAnswer }) {
+  if (correctAnswer > 3) {
+    return (
+      <div className="celebration_image-component">
+        <img src={emojiConfettiBall} alt="emoji-confetti-ball.svg" />
```

Comment:
Same goes with the emojiConfettiBall image as well

Reply:
Yes ...
But I think if you test it on your comupter, mabye you still have old path of images ... because Gofran change the name folder from SVG to icons..... check it please

Reply:
Same with this image, flex issue.